api/src/main/resources/moduleApplicationContext.xml

```diff
@@ -89,6 +88,34 @@
     <bean id="registrationcore.mpiPatientSearcher"
           class="org.openmrs.module.registrationcore.api.mpi.openempi.OpenEmpiPatientsSearcher"/>
 
+    <!-- PixPdq Mpi implementation part -->
+    <bean id="registrationcore.mpi.implementation.PixPdq"
```

Comment:
Is there any reason you are using xml and not annotations? This file is already fairly long, would be nice to avoid adding to it when necessary.

Reply:
So what you are suggesting is to delete all these MPI and PIXPDQ MPI bean declarations and in each of these classes, use the annotation @ Bean  as per https://docs.spring.io/spring-javaconfig/docs/1.0.0.M4/reference/html/ch02s02.html ?

Reply:
I believe it's the correct thing to do going forward, I'd wait to remove them and first confirm with others that they are okay with us doing this, I expect no opposition, I just like to keep others informed.

Reply:
Sorry! actually what I meant by the first comment was to annotate the classes with @Component rather than adding bean entries for them in this xml file just to keep it shorter

Reply:
I guess it's not a big deal keeping them here, file isn't as long as I thought